rom/dev/README.md

```diff
@@ -385,25 +401,34 @@ Local Device ID Layer derives the Owner CDI & ECC Keys. This layer represents th
 	`LDevIdTbsDigest = sha384_digest(LDevIdTbs)`
 	`Result = ecc384_verify(LDevIdPubKey, LDevIdTbsDigest, LDevIdCertSig)`
 
-9.	Store and lock (for write) the LDevID Certificate Signature in the sticky Data Vault (48 bytes) Slot 2 & Slot 3
+9.	Store and lock (for write) the LDevID Certificate Signature in the sticky Data Vault (48 bytes) Slot 0 & Slot 1
 
-	`dv48_store(LDevIdCertSig.R, Dv48Slot2)`
-    `dv48_lock_wr(Dv48Slot2)`
-    `dv48_store(LDevIdCertSig.S, Dv48Slot3)`
-    `dv48_lock_wr(Dv48Slot3)`
+	`dv48_store(LDevIdCertSig.R, Dv48Slot0)`
+    `dv48_lock_wr(Dv48Slot0)`
+    `dv48_store(LDevIdCertSig.S, Dv48Slot1)`
+    `dv48_lock_wr(Dv48Slot1)`
 
 **Post-Conditions:**
 * Vault state as follows:
 
 | Slot | Key Vault | PCR Bank | Data Vault 48 Byte (Sticky) | Data Vault 4 Byte (Sticky) |
 |------|-----------|----------|-----------------------------|----------------------------|
-| 0 | | | 🔒LDevID Pub Key X |
-| 1 | | | 🔒LDevID Pub Key Y |
-| 2 | | | 🔒LDevID Cert Signature R |
-| 3 | | | 🔒LDevID Cert Signature S |
+| 0 | | | 🔒LDevID Cert Signature R |
+| 1 | | | 🔒LDevID Cert Signature S |
+| 2 | | | 🔒LDevID Pub Key X |
+| 3 | | | 🔒LDevID Pub Key Y |
 | 5 | LDevID Private Key (48 bytes) |
 | 6 | LDevID CDI (48 bytes) |
 
+### 9.6 Handling commands from Mailbox
+ROM supports the following set of commands before handling the FW_DOWNLOAD command (described in section 9.6). Once the FW_DOWNLOAD is issued, ROM stops processing any additional mailbox commands.
+1. **STASH_MEASUREMENT**: Upto eight measurements can be sent to the ROM for recording. Format of a measurement is documented at https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/README.md#stash_measurement
+2. **VERSION**: [TODO] Add links to data structure formats once available.
```

Comment:
provide a reference to documentation of the rest of these commands?

Reply:
The documentation is not available as of now. Adding TODOs to update this once it is added to https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/README.md